client/components/site-selector/style.scss

```diff
@@ -29,6 +29,21 @@
 	&.is-large .site-selector__sites {
 		border-top: 1px solid var(--color-neutral-10);
 	}
+
+	.search-component {
+		.search-component__input {
+			width: 100%;
+		}
```

Comment:
This prevents the horizontal scrolling and clipping of the X icon of the search component inside the site selector.